sources/documents/2_RTB/PianoDiQualifica/registro_modifiche.tex

```diff
@@ -23,7 +23,8 @@ \section*{Registro delle modifiche}
 		\endlastfoot
 
 		% IN ORDINE DALLA MODIFICA PIÙ RECENTE ALLA PIÙ VECCHIA
-		0.0.2 & 2024-05-15 & \martina & \sebastiano & Inserimento tabelle delle metriche \\
+		0.0.3 & 2024-06-03 & \sebastiano & \riccardo, \raul, \marco & Aggiornamento metriche \\
```

Comment:
Forse anche in questo documento si deve avanzare di versione in `model/global.tex`